gitpod-io/gitpod
# Org SSO Page updates

## Description

Makes various updates to the Org Settings => SSO page.

* Adds Delete support for OIDC clients
* Adds Edit support to the UI for OIDC clients (api needs implementing still)
* Updates page to use react-query
* Updates page to take advantage of existing components similar to Git Auth.

## Related Issue(s)

Relates to #15961

## How to test

**Preview Env:** https://bmh-org-se0fee9b85fb.preview.gitpod-dev.com/workspaces
* Create an Org, visit the SSO page
* Create a client.
* Try to edit it. UI should populate w/ client data, but saving should product an error as the API isn't implemented yet.
* Try to delete the client

## Release Notes

```release-note
NONE
```

## Documentation


## Build Options:

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
